1/incrementing-the-value.md

```diff
@@ -43,7 +43,7 @@ impl MyContract {
 }
 ```
 
-However, if the value is not initialized, your contract will compile fine, but will panic during contract execution! **We really cannot understate how easy it is to make mistakes this way.**
+However, if the value is not initialized, your contract will compile fine, but will panic during contract execution! **We really cannot overstate how easy it is to make mistakes this way.**
```

Comment:
Now that I think about it, it seems to be actually correct. I think I got it backwards.